tools/bin/3d-tiles-tools.js

```diff
@@ -250,6 +257,29 @@ function readGlbWriteB3dm(inputPath, outputPath, force) {
         });
 }
 
+function readGlbWriteI3dm(inputPath, outputPath, force) {
+    outputPath = defaultValue(outputPath, inputPath.slice(0, inputPath.length - 3) + 'i3dm');
+    return fileExists(outputPath)
+        .then(function(exists) {
+            if (!force && exists) {
+                console.log('File ' + outputPath + ' already exists. Specify -f or --force to overwrite existing file.');
+                return;
+            }
+            return fsReadFile(inputPath)
+                .then(function(data) {
+                    // Set I3dm spec requirements
+                    var featureTable = { INSTANCES_LENGTH : 0,
+                                         EAST_NORTH_UP : true,
+                                         POSITION : { byteOffset : 0 }
+                                       };
```

Comment:
Usually we format JSON in-line with the rest of the code, like:

```javascript
var featureTable = {
    INSTANCES_LENGTH : 0,
    EAST_NORTH_UP : true,
    POSITION : {
        byteOffset : 0
    }
};
```
Good call on the EAST_NORTH_UP.